--- Makefile.in	2008-05-17 20:36:44.000000000 +0200
+++ Makefile.in.oden	2009-03-07 15:36:48.000000000 +0100
@@ -11,6 +11,7 @@
 INSTALL_PROGRAM = @INSTALL_PROGRAM@
 LDFLAGS = @LDFLAGS@
 LIBS = @LIBS@
+DESTDIR = 
 
 RM = rm -f
 MKINSTALLDIRS = mkdir -p
@@ -47,34 +48,34 @@
 
 install-conf:
 	@echo Installing configuration file  ;
-	@if [ ! -d $(prefix)/squidGuard ]; then \
-		$(MKINSTALLDIRS) $(prefix)/squidGuard ; \
-		echo Created directory $(prefix)/squidGuard ; \
-		chown -R $(SQUIDUSER) $(prefix)/squidGuard || exit 1  ; \
-		echo Assigned $(prefix)/squidGuard to user $(SQUIDUSER) ; \
+	@if [ ! -d $(DESTDIR)$(prefix)/squidGuard ]; then \
+		$(MKINSTALLDIRS) $(DESTDIR)$(prefix)/squidGuard ; \
+		echo Created directory $(DESTDIR)$(prefix)/squidGuard ; \
+		chown -R $(SQUIDUSER) $(DESTDIR)$(prefix)/squidGuard || exit 1  ; \
+		echo Assigned $(DESTDIR)$(prefix)/squidGuard to user $(SQUIDUSER) ; \
 	fi ; 
-	@if [ ! -d $(dbhomedir) ]; then \
-		$(MKINSTALLDIRS) $(dbhomedir) ; \
-		echo Created directory $(dbhomedir) ; \
-		chown -R $(SQUIDUSER) $(dbhomedir) || exit 1 ; \
-		echo Assigned $(dbhomedir) to user $(SQUIDUSER) ; \
+	@if [ ! -d $(DESTDIR)$(dbhomedir) ]; then \
+		$(MKINSTALLDIRS) $(DESTDIR)$(dbhomedir) ; \
+		echo Created directory $(DESTDIR)$(dbhomedir) ; \
+		chown -R $(SQUIDUSER) $(DESTDIR)$(dbhomedir) || exit 1 ; \
+		echo Assigned $(DESTDIR)$(dbhomedir) to user $(SQUIDUSER) ; \
 	fi ; 
-	@if [ ! -d $(logdir) ]; then \
-		$(MKINSTALLDIRS) $(logdir) ; \
-		echo Created directory $(logdir) ; \
-		chown -R $(SQUIDUSER) $(logdir) || exit 1 ; \
-		echo Assigned $(logdir) to user $(SQUIDUSER) ; \
+	@if [ ! -d $(DESTDIR)$(logdir) ]; then \
+		$(MKINSTALLDIRS) $(DESTDIR)$(logdir) ; \
+		echo Created directory $(DESTDIR)$(logdir) ; \
+		chown -R $(SQUIDUSER) $(DESTDIR)$(logdir) || exit 1 ; \
+		echo Assigned $(DESTDIR)$(logdir) to user $(SQUIDUSER) ; \
 	fi ; 
 	@if [ ! -d `dirname $(configfile)` ]; then \
 		umask 022 ; \
-		mkdir -p `dirname $(configfile)` ; \
-		echo No configuration directory found. Created `dirname $(configfile)`. ; \
+		mkdir -p $(DESTDIR)`dirname $(configfile)` ; \
+		echo No configuration directory found. Created $(DESTDIR)`dirname $(configfile)`. ; \
 	fi;
 	@if test ! -f $(configfile); then \
-		cp samples/sample.conf $(configfile) || exit 1  ; \
+		cp samples/sample.conf $(DESTDIR)$(configfile) || exit 1  ; \
 		echo Copied sample squidGuard.conf ; \
-		chmod 644 $(configfile) || exit 1 ; \
-		echo $(configfile) is now readable ; \
+		chmod 644 $(DESTDIR)$(configfile) || exit 1 ; \
+		echo $(DESTDIR)$(configfile) is now readable ; \
 		echo The initial configuration is complete. ; \
 	else \
 		echo Configuration file found. Not changing anything ; \
@@ -85,10 +86,10 @@
 
 install-build:
 	@echo Installing squidGuard 
-	@if [ ! -d $(bindir) ]; then \
-		$(MKINSTALLDIRS) $(bindir) ; \
+	@if [ ! -d $(DESTDIR)$(bindir) ]; then \
+		$(MKINSTALLDIRS) $(DESTDIR)$(bindir) ; \
 	fi ; \
-	cp src/squidGuard $(bindir) || exit 1 ;  \
+	cp src/squidGuard $(DESTDIR)$(bindir) || exit 1 ;  \
 	echo Done. ;
 
 clean::
