--- squidGuard-1.4/src/sg.l	2009-11-23 05:04:09.000000000 +0100
+++ squidGuard-1.4.dlucio/src/sg.l	2009-11-23 05:01:30.000000000 +0100
@@ -189,7 +189,7 @@
                    }
 <STRING_STATE>.    { if (string_buf_ptr-string_buf < sizeof(string_buf)-1) *string_buf_ptr++ = yytext[1];  }
 
-[a-zA-Z\/][a-zA-Z0-9/_\-\.\/\:\%\+\?=&]*  {yylval.string = yytext ; return WORD;}
+[a-zA-Z0-9\/][a-zA-Z0-9/_\.\/\:\%\+\?=&!&\(\)]*  {yylval.string = yytext ; return WORD;}
 s@(([^@}]|\\@|\\\})+)@(([^@}]|\\@|\\\})+)@[girR]* {yylval.string = yytext; return SUBST;}
 [ \t]+     ;
 \n        {lineno++;}
